### PR TITLE
Indentation rules

### PR DESCRIPTION
**Pull request by apurvis**
@dtboctor @jz042 @ryanfoxsquire

### CODE REVIEW

File: _includes/sqlstyle.guide.md [hunk 1]
```diff
@@ -277,27 +291,30 @@ FROM my_tmp_table
 
 #### Sub-queries
 
-In PostgreSQL you should probably be doing subqueries with `WITH` clauses.
+In PostgreSQL you should be doing subqueries with `WITH` clauses and avoiding the use of inline
+subqueries.
```
Comment (dtboctor) [inline _includes/sqlstyle.guide.md]: Do you have a justification for this other than it's your preference?  I don't think this is always readable for subqueries that have meaningless data.

**Comment by dtboctor** [comment]
I hate to say this, but this style guide has gone from a basic good idea about how some indentation should work to make queries readable to a literal waste of time and money.

File: _includes/sqlstyle.guide.md (hunk 1 above)
Comment (apurvis) [inline _includes/sqlstyle.guide.md]: I think it's more readable but people can disagree, or we can switch it back to "probably".  [your recent PR|https://github.com/lumoslabs/lumos_data_warehouse/pull/3627/files] provides a good case study for people confused about what they are voting on

Comment (slpsys) [inline _includes/sqlstyle.guide.md]: Yeah, i'd support this PR if it were still "probably"

**Comment by apurvis** [comment]
not sure what you mean by that; people's queries in aleph have gotten extremely noticeably more uniform and easier to read since the style guide which only took like 1-2 hours to put together.

anyways i changed the subquery section to be "probably" so i'm merging this.

**Pull request merged**